devdoc/worker_thread_requirements.md

```diff
@@ -47,9 +47,7 @@ MOCKABLE_FUNCTION(, WORKER_THREAD_HANDLE, worker_thread_create, WORKER_FUNC, wor
 
 - **SRS_WORKER_THREAD_01_037: [** `worker_thread_create` shall create a state manager object by calling `sm_create` with the name `worker_thread`. **]**
 
-- **SRS_WORKER_THREAD_01_006: [** `worker_thread_create` shall create an auto reset unnamed event used for signaling the thread to shutdown by calling `CreateEvent`. **]**
-
-- **SRS_WORKER_THREAD_01_007: [** `worker_thread_create` shall create an auto reset unnamed event used for signaling that a new execution of the worker function should be done by calling `CreateEvent`. **]**
+- **SRS_WORKER_THREAD_01_006: [** `worker_thread_create` shall initialize the internal objects. **]**
```

Comment:
> internal objects [](http://example.com/codeflow?start=77&length=17)

cannot write unit tests that test "internal objects" without those being specified what they are. Here it is needed a more concise phrasing.